src/docs/metrics/ethereum-fees/index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: Ethereum fees
+author: Filip
+date: 2024-03-01
+description: Ethereum fees
+---
+
+## Description
+Gas fees are the fees paid for the computational efforts required to execute a transaction or 
+operation within a smart contract on the Ethereum network. Each operation in a transaction consumes 
+a specific amount of gas.
+The total gas fee you pay consists of several components:
+- Base fee: This is a mandatory fee established by the network for processing a transaction.
+- Priority fee: An additional, voluntary payment intended to motivate node operators to prioritize 
+your transaction.
+- Units of gas used: Gas signifies computational effort. More complex tasks, such as interacting 
+with a smart contract, require more gas compared to simpler actions like sending a transaction.
+
+The base fee is computed independently of the current block and instead relies on the preceding blocks, 
+enhancing predictability for users regarding transaction fees. Upon block creation, this base fee 
+is burned, effectively removing it from circulation.
+
+
+Ethereum fees metrics:
+- `fees`: Total fees paid in ETH over a daily interval
+- `fees_intraday`: Total fees paid in ETH over 5-minute time intervals
+- `fees_usd`:  Total fees paid in USD over a daily interval
+- `fees_usd_intraday`: Total fees paid in USD over 5-minute time intervals
+
+
+- `average_fees_usd`: Average fees paid in USD over a daily interval
+- `average_fees_usd_5m`: Average fees paid in USD over 5-minute time intervals
+- `median_fees_usd`: Median fees paid in USD over a daily interval
+- `median_fees_usd_5m`: Median fees paid in USD over 5-minute time intervals
+
+
+- `fees_burnt_5m`: Fees burnt (removed from circulation) in ETH within a 5-minute interval
+- `fees_burnt_usd_5m`: Fees burnt (removed from circulation) in USD within a 5-minute interval
+
+
+- `fees_to_network_circulation_usd_1d`: Calculated using the formula: `fees * daily_avg_price_usd / network_circulation_usd_1d`
```

Comment:
maybe we should explain that `network_circulation_usd_1d` is the $ value of all coins moved on ethereum that we track.

Reply:
Fixed